packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -235,6 +237,23 @@ export class Account {
   async utxosToAccount (payload: UtxosToAccountPayload, utxos: UtxosToAccountUTXO[] = []): Promise<string> {
     return await this.client.call('utxostoaccount', [payload, utxos], 'number')
   }
+
+  /**
+   * Returns count of account history
+   *
+   * @param {OwnerType | string} [owner= OwnerType.MINE] single account ID (CScript or address) or reserved words 'mine' to list history count for all owned accounts or 'all' to list whole DB
+   * @param {AccountHistoryCountOptions} [options]
+   * @param {boolean} [options.no_rewards]  Filter out rewards
+   * @param {string} [options.token]  Filter by token
+   * @param {string} [options.txtype]  Filter by transaction type, supported letter from 'CRTMNnpuslrUbBG'
+   * @return {Promise<number>}  Count of account history
+   */
+  async accountHistoryCount (
```

Comment:
```suggestion
  async historyCount (
```

Everything in this class is account, `account.accountHistoryCount` double semantic. shorten to `account.historyCount` for brevity.